dev/io.openliberty.org.eclipse.microprofile/health.3.1.bnd

```diff
@@ -0,0 +1,26 @@
+#*******************************************************************************
+# Copyright (c) 2021 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#    IBM Corporation 
+# This wraps the external JAR.
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+Bundle-SymbolicName: io.openliberty.org.eclipse.microprofile.health.3.1; singleton:=true
+
+Import-Package: \
+  *
+
+Export-Package: org.eclipse.microprofile.health;version=3.1, \
+                org.eclipse.microprofile.health.spi;version=3.1
```

Comment:
This should have been `1.1` but we can't change it now because the previous version was already `3.0` and we can't go backwards.
https://github.com/eclipse/microprofile-health/blob/master/api/src/main/java/org/eclipse/microprofile/health/spi/package-info.java